packages/fiber/src/core/loop.ts

```diff
@@ -2,7 +2,7 @@ import * as THREE from 'three'
 import { Root } from './renderer'
 import { RootState, Subscription } from './store'
 
-type GlobalRenderCallback = (timeStamp: number) => void
+export type GlobalRenderCallback = (timestamp: number) => void
 type SubItem = { callback: GlobalRenderCallback }
 
 function createSubs(callback: GlobalRenderCallback, subs: Set<SubItem>): () => void {
```

Comment:
This should be backported to v8 in a patch release. We didn't actually export anything in #2481.